### DISCUSSION TITLE
Is it normal that if n_lags + n_forecasts is bigger in length then the validation set, neuralprophet throw out an "Index out of range" error during training?

### DISCUSSION THREAD

**Discussion by Domics10** [ourownstory/neural_prophet#1481] [General]
**Prerequisites**

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in a new virtualenv with only neuralprophet installed, directly from github:
      ```shell
      git clone <copied link from github>
      cd neural_prophet
      pip install .
      ```
    * Checked the Answered Questions on the Github Discussion board: https://github.com/ourownstory/neural_prophet/discussions
      If you have the same question but the Answer does not solve your issue, please continue the conversation there.
    * Checked that your issue isn't already filed: https://github.com/ourownstory/neural_prophet/issues
      If you have the same issue but there is a twist to your situation, please add an explanation there.
    * Considered whether your bug might actually be solvable by getting a question answered: 
      * Please [post a package use question](https://github.com/ourownstory/neural_prophet/discussions/categories/q-a-get-help-using-neuralprophet)
      * Please [post a forecasting best practice question](https://github.com/ourownstory/neural_prophet/discussions/categories/q-a-forecasting-best-practices)
      * Please [post an idea or feedback](https://github.com/ourownstory/neural_prophet/discussions/categories/ideas-feedback)

**Describe the bug**

**To Reproduce**

1. With the data '...'
2. Setting the model hyperparameters '....'
3. After running these code lines '....'
4. When using this function '....'
5. See error

**Expected behavior**

**What actually happens**

Describe what happens, and how often it happens.

**Screenshots**

If applicable, add screenshots and console printouts to help explain your problem.

**Environment (please complete the following information):**

 - Python environment [e.g. Python 3.8, in standalone venv with no other packages]
 - NeuralProphet version and install method [e.g. 2.7, installed from PYPI with `pip install neuralprophet`]

**Additional context**
